File: app/src/main/java/com/example/builday365/Model/RoomDb.java
package com.example.builday365.Model;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import androidx.room.TypeConverters;

import com.example.builday365.Model.Timeline.Converters;
import com.example.builday365.Model.Timeline.TimeLine;
import com.example.builday365.Model.Timeline.TimeLineDao;

@Database(entities = {TimeLine.class}, version = 1)
@TypeConverters({Converters.class})
public abstract class RoomDb extends RoomDatabase {
    public abstract TimeLineDao timeLineDao();

}

File: app/src/main/java/com/example/builday365/model/RoomDb.kt
package com.example.builday365.model

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters
import com.example.builday365.model.test.Converters
import com.example.builday365.model.test.TimeLine
import com.example.builday365.model.test.TimeLineDao

@Database(entities = [TimeLine::class], version = 1)
@TypeConverters([Converters::class])
abstract class RoomDb : RoomDatabase() {
    abstract fun timeLineDao(): TimeLineDao?
}